doc/Manuals/ug-ss_x-road_6_security_server_user_guide.md

```diff
@@ -177,7 +177,11 @@ Doc. ID: UG-SS
   * [8.2 Displaying and Changing the Members of a Local Group](#82-displaying-and-changing-the-members-of-a-local-group)
   * [8.3 Changing the description of a Local Group](#83-changing-the-description-of-a-local-group)
   * [8.4 Deleting a Local Group](#84-deleting-a-local-group)
-* [9 Communication with the Client Information Systems](#9-communication-with-the-client-information-systems)
+* [9 Communication with Information Systems](#9-communication-with-information-systems)
+  * [9.1 Communicating with security server as a service consumer](#91-communicating-with-security-server-as-a-service-consumer)
+  * [9.2 Communicating with security server as a service provider](#92-communicating-with-security-server-as-a-service-provider)
```

Comment:
I would change this to "Communication with Service Provider Information Systems" to make it more clear. Also, headings should be capitalized - all words are capitalized unless they are conjunctions, articles, or prepositions.